src/macaron/slsa_analyzer/git_url.py

```diff
@@ -390,8 +390,10 @@ def get_repo_full_name_from_url(url: str) -> str:
     return full_name
 
 
-def get_remote_url_of_local_repo(git_obj: Git) -> str:
-    """Get the origin remote url of a repository.
+def get_remote_origin_of_local_repo(git_obj: Git) -> str:
+    """Get the origin remote of a repository.
+
+    Note that this origin remote can be either a remote url to point to a local repo.
```

Comment:
```suggestion
    Note that this origin remote can be either a remote url or a path to point to a local repo.
```
?

Reply:
I think I forgot the `or` here. How about updating it to:
` Note that this origin remote can be either a remote url or a path to a local repo.`?

Reply:
> I think I forgot the `or` here. How about updating it to: ` Note that this origin remote can be either a remote url or a path to a local repo.`?

Sounds good.

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/153/commits/bdeba388db589ff62f060dbb87c5512bb05ba587